### PR TITLE
optimizations

### PR DESCRIPTION
**Pull request by Aws0mee**
Made some more performance fixes.

### CODE REVIEW

**Comment by DrVrej** [comment]
If possible, please update this request to the revamp version, thank you!

**Comment by Aws0mee** [comment]
> If possible, please update this request to the revamp version, thank you!

Will do

File: lua/weapons/gmod_tool/stools/vjstool_npcmover.lua
Comment (DrVrej) [inline lua/weapons/gmod_tool/stools/vjstool_npcmover.lua:1]: This change still applies, will be included in the next commit!

File: lua/weapons/gmod_tool/stools/vjstool_npcspawner.lua
Comment (DrVrej) [inline lua/weapons/gmod_tool/stools/vjstool_npcspawner.lua:1]: These changes still apply, will be included in the next commit!

**Review by DrVrej — COMMENTED** [review]
Going through each change against the revamp update, there was still 3 changes from this PR that applied, they included in the latest commit 301a46e3e912bae979ad47691f6343c5ee5f73f1 Thanks again for these suggestions!

**Pull request closed without merging**